File: patterns/javascript-patterns/pattern110.js
// Pattern 110: ...

//         *
//       * * *
//     *   *   *
//   *     *     *
// * * * * * * * * *
//   *     *     *
//     *   *   *
//       * * *
//         *


let n = 9;
let c1 = (n - 1) / 2;
let c2 = 3 * n / 2 - 1;

for (let i = 0; i < n; i++) {
    let output = '';
    for (let j = 0; j < n; j++) {
        if (i + j === c1 || i - j === c1 || j - i === c1 || i + j === c2 || i === c1 || j === c1) {
            output += "* ";
        } else {
            output += "  ";
        }
    }
    console.log(output);
}

File: patterns/python-patterns/pattern110.py
# Pattern 110: ...

#         *
#       * * *
#     *   *   *
#   *     *     *
# * * * * * * * * *
#   *     *     *
#     *   *   *
#       * * *
#         *

n = 9
c1 = (n - 1) // 2
c2 = 3 * n // 2 - 1

for i in range(n):
    for j in range(n):
        if i + j == c1 or i - j == c1 or j - i == c1 or i + j == c2 or i == c1 or j == c1:
            print("* ", end="")
        else:
            print("  ", end="")
    print()
